src/cloud-cli/meltano/cloud/cli/project.py

```diff
@@ -296,3 +314,27 @@ async def use_project(
         ),
         fg="green",
     )
+
+
+@project_group.command("add")
+@click.argument("project_name", type=str, required=True)
+@click.argument("git_repository", type=str, required=True)
```

Comment:
Can we make this a required option instead of a positional argument? That way it's easier to change in the future.

The project name may still be acceptable as a positional argument, but we could consider making it an option too.

Wdyt?

https://clig.dev/#arguments-and-flags
https://medium.com/@jdxcode/12-factor-cli-apps-dd3c227a0e46